src/components/book/book.js

```diff
@@ -24,25 +25,53 @@ const Book = (props) => {
   };
   return (
   // return jsx syntax for displaying in html
-    <div id={id}>
-      <div>
-        <h2>{title}</h2>
-        <p>{author}</p>
+    <li className="styNone">
+      <div className="book">
+        <div className="book-content">
+          <div className="book-info">
+            <h4 className="book-category">Action</h4>
+            <h2 className="book-title" id={id}>{title}</h2>
+            <h6 className="book-author">{author}</h6>
+            <div className="action-buttons">
+              <button className="button-outline" type="button">Comments</button>
+              <div className="vertical-divider" />
+              <button
+                className="button-outline"
+                type="button"
+                onClick={() => {
+                  filterBooks(id);
+                  dispatch(removeBook(id));
+                }}
+              >
+                Remove
+              </button>
+              <div className="vertical-divider" />
+              <button className="button-outline" type="button">Edit</button>
+            </div>
+          </div>
+          <div className="progress-container">
+            <div className="circular-progress-container">
+              <div className="circular-progress" />
+            </div>
+            <div className="progress-state">
+              <p className="percent-complete">75%</p>
+              <p className="completed">Completed</p>
+            </div>
+            <span className="progress-divider" />
+            <div className="current-chapter-container">
+              <div>
+                <p className="chapter-label">CURRENT CHAPTER</p>
+                <p className="chapter">Chapter 17</p>
+              </div>
+              <div>
+                <button className="primary-button" type="button">UPDATE PROGRESS</button>
+              </div>
+            </div>
+          </div>
+        </div>
```

Comment:
- [ ] You are required to replicate the design exactly. Kindly use the correct font for the buttons, links, and other details in this book component. Only the progress circle and the "Action" seem to have the correct font. Consider using the design as a reference.

  Maybe your Google font is not being loaded properly?

  Your Design | Expected Design
  :---:|:---:
  ![udud](https://user-images.githubusercontent.com/54848363/208054159-094f2da9-b69d-4ac7-9695-0871a4b89e51.jpg) | ![uddu](https://user-images.githubusercontent.com/54848363/208054147-c1828d03-6aa4-4a22-aa6d-bf4fc0d71049.jpg)